payment-intent/cs/payment-intent-auto.cs

```diff
@@ -0,0 +1,16 @@
+var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
+var result = await paymentIntentsClient.CreatePaymentIntentAsync(new CreatePaymentIntentRequest
+{
+    Amount = new()
+    {
+        Value = checkoutRequest.Amount,
```

Comment:
please replace checkoutRequest.Amount with actual values, like in other samples

Reply:
Removed checkoutRequest usages.